firecracker/microvm.py

```diff
@@ -278,6 +284,34 @@ def enable_rootfs(self, path_on_host: str) -> str:
         else:
             return path_on_host
 
+    def mount_rootfs(self, path_on_host: str) -> str:
+        """Mount a rootfs to the VM.
+        """
+        self.mounted_fs = path_on_host
+        if self.use_jailer:
+            rootfs_filename = Path(path_on_host).name
+            jailer_path_on_host = f"{DEVICE_BASE_DIRECTORY}/{rootfs_filename}"
+            if not check_device_exists(Path(f"{self.jailer_path}/{jailer_path_on_host}")):
+                device_vm_path = f"{DEVICE_BASE_DIRECTORY}/{self.vm_id}"
+                subprocess.run(["mkdir", "-p", device_vm_path],
+                               check=True,
+                               capture_output=True)
+                path_to_mount = f"{device_vm_path}/{rootfs_filename}"
+                os.symlink(path_on_host, path_to_mount)
+                os.system(f"cp -vap /dev/dm* {self.jailer_path}/dev/")
```

Comment:
Why mix `os.system` with `subprocess.run` ?

Reply:
Because the `os.system` doesn't return any output, but `subprocess.run` does, and sometimes we don't want to check the system output.